packages/workers-editor-shared/package.json

```diff
@@ -42,6 +42,7 @@
 		"eslint": "^8.57.1",
 		"react": "^18.3.1",
 		"react-dom": "^18.3.1",
+		"typescript": "catalog:default",
```

Comment:
fixes typescript errors for me when building - previously it was trying to use a non-installed ts ver